tests/test_architectures.py

```diff
@@ -22,7 +25,8 @@ def test_embedding_models_return_normalized_embeddings():
     that are L2-normalized.
     """
     for architecture in Architecture:
-        embedding_model: tf.keras.Model = architecture.get_embedding_model()
+        embedding_model: tf.keras.Model = architecture.get_embedding_model(
+        ).model
```

Comment:
Je kunt hier als het goed is ook het volgende van maken:

```python
model = architecture.get_model()
batch_input_shape = model.input_shape
```
Dat is wat netter.

Reply:
Klopt, aangepast.